JoinColony/colonyCDapp
# Fix: Use subscriptions to  automatically show Advanced Payment actions on the Actions Table

## Description

This PR addresses an issue where Advanced Payments are not automatically being shown on the Actions table.

At first, I tried to simply subscribe to action creation. But I found that this was not the best solution since the action's metadata isn't yet ready, which causes our activity table to use fallback action descriptions which is not ideal behaviour.

So instead, I'm subscription to an action's metadata creation which works perfectly ✨ 

![AP staking demo](https://github.com/user-attachments/assets/0f3b7d6b-675a-473f-8f91-df3a03ec448e)

## Testing

> [!NOTE]
> If you're not seeing the advanced payment on the actions table, first verify that you're not able to see any new action types. If you're unable to see new actions then something probably broke in your environment.

1. Bring up the Advanced Payment form
2. Fill in all required fields
3. Submit the form
4. Verify that the Latest Activity table is:
- automatically updated
- using the correct title you've used for your action 

Resolves #3581
